salt/modules/netbox.py

```diff
@@ -31,7 +31,7 @@
 
 try:
     import pynetbox
-    from pynetbox.lib import RequestError
+    from pynetbox.core.query import RequestError
```

Comment:
I'd probably do something like `from pynetbox import RequestError` so any path changes in the future don't end up breaking it again, or better yet just catch `pynetbox.RequestError`.

Reply:
@misch42 Can you revisit this one and address the suggestions by @zachmoody?